### ISSUE TITLE
error shared libraries: libssl.so

### ISSUE THREAD

**Issue by tiagofrancafernandes** [vercel-community/php#566]
# Question

All builds on Vercel returns the error:

```log
php: error while loading shared libraries: libssl.so.10: cannot open shared object file: No such file or directory
```

![image](https://github.com/user-attachments/assets/9423862a-3770-4c71-bca8-ac34bc708d4d)

**Comment by Mashiane** [comment]
Same here, i cant get it to work. Hopefully we will get a solution.

**Comment by nadar** [comment]
same :point_up:

**Comment by thedoggybrad** [comment]
change your node.js version guys to 18.x, it will work. Make sure to edit a file on your repo or source after applying the changes so that it can redeploy.
@nadar @Mashiane @tiagofrancafernandes

**Comment by f3l1x** [comment]
I am doing my best to fix support of newer Node.js. I will keep you in loop.

**Comment by f3l1x** [comment]
Hi, it should be fixed, can you try version `0.7.2`?

https://example-php-8-3.vercel.app/
